project/src/main/webapp/events-feed.js

```diff
@@ -1,15 +1,20 @@
-$(async function() {
-  const allEvents = await getAllEvents();
-  populateAllEvents(allEvents);
+$(function() {
+  populateEvents();
 });
 
```

Comment:
javadoc?

Reply:
With this function already being notated as a `document.ready` function, I don't think it would be helpful to add a comment like "function runs once document is ready" or "calls first set of functions for document"